### PR TITLE
#1219 Fix for missing request in JWTGuard

### PR DESCRIPTION
**Pull request by michielpeeters**
Fixed JWTGuard when it tries to parse the authorization header

### CODE REVIEW

File: src/JWTGuard.php [hunk 1]
```diff
@@ -72,6 +72,8 @@ public function user()
             return $this->user;
         }
 
+        $this->jwt->parser()->setRequest($this->request);
+        
         if ($this->jwt->getToken() &&
```
Comment (tymondesigns) [inline src/JWTGuard.php]: It would be better to change it to 
```php
if ($this->jwt->setRequest($this->request)->getToken() && // etc
```

**Review by tymondesigns — CHANGES_REQUESTED** [review]

File: src/JWTGuard.php (hunk 1 above)
Comment (michielpeeters) [reply src/JWTGuard.php]: Agree, I have it changed in a new commit :-)

**Comment by tymondesigns** [comment]
Just need to update tests and fix style and then I'm happy to merge this

**Comment by tymondesigns** [comment]
@michielpeeters Are you comfortable tweaking the tests for this?

**Comment by tymondesigns** [comment]
#1243 now implements this.

Thanks for the effort @michielpeeters

**Pull request closed without merging**